app/actionsRenderingEngine/conf/gazebo/config.ini

```diff
@@ -0,0 +1,98 @@
+[vision]
+period                              20
+minBufSize                          10
+timeTol                             0.5
+varThresh                           5.0
+dominant_eye                        left
+raw_detection_wait_thresh           15.0
+motion_detection_wait_thresh        5.0
+object_detection_wait_thresh        5.0
+
+[motor]
+robot                               icubSim
+part_used                           both_arms
+actionprimitives_layer              2
+waveing                             off
+thread_period                       30
+default_exec_time                   0.75
+reach_tol                           0.01
+torso_pitch                         on
+torso_roll                          off
+torso_yaw                           on
+torso_pitch_min                     0.0
+torso_pitch_max                     50.0
+tracking_mode                       off
+verbosity                           on
+calib_kin_gain                      0.005
+target_in_range_thres               0.6
+stereo_track                        off
+dominant_eye                        left
+actions                             actions
+eyes_traj_time                      0.4
+neck_traj_time                      0.7
+home_fixation_point                 (xyz -0.3 0.0 0.0)
+reach_above_displacement            (0.0 0.0 0.15)
+grasp_above_relief                  (0.0 0.0 0.035)
+push_above_relief                   (0.0 0.0 0.03)
+go_up                               0.1
+table_height_tolerance              0.05
+net                                 network.ini
+exploration_poses                   exploration_poses.ini
+hand_sequences_file                 gazebo/hand_sequences.ini
+stereo_kp                           0.007
+stereo_ki                           0.002
+stereo_kd                           0.0
+neck_pitch_range                    (-45.0 0.0)
+neck_roll_range                     (0.0)
+impedance                           off
+touch_threshold                     2.0
+impedence_arm_stiffness             (0.4 0.4 0.4 0.2 0.05)
+impedence_arm_damping               (0.03 0.03 0.03 0.01 0.0)
+impedence_torso_stiffness           (0.0 0.9 0.9)
+impedence_torso_damping             (0.06 0.06 0.06)
+arm_target_tol                      0.01
+grasp_model_type                    tactile
```

Comment:
`grasp_model_type` should be set to `springy`, otherwise the module looks for `/icubSim/skin/left_hand`.

Reply:
Right! I'll do that, thanks!